content/blog/2021/09/2021-09-21-jenkins-at-devops-world.adoc

```diff
@@ -0,0 +1,78 @@
+---
+:layout: post
+:title: "Join Jenkins at DevOps World 2021"
+tags:
+- events
+- conference
+- cicd
+- talks
+- workshops
+author: alyssat
+description: >
+  DevOps World 2021 will include significant Jenkins workshops, sessions, and interactions.
+  Join us September 28-30, 2021 for Jenkins workshops, presentations, and conversations.
+opengraph:
+  image: /images/conferences/devops-world-2021.png
+links:
+  discourse: true
+---
+
+link:https://www.devopsworld.com/[DevOps World] has been the largest gathering for Jenkins for many years.
+In keeping with tradition, many Jenkins presentations and sessions are planned for this year's event.
+
+image:/images/conferences/devops-world-2021.png[role="center", align="center", link="https://www.devopsworld.com/"]
+
+Join us for DevOps World on September 28 - 30, 2021.
```

Comment:
Do you want to mention the time and timezones?  I think I heard that some sessions will be done twice to accomodate a few more timezones.
Also that registered users can view the sessions, including chat comments, after the session if the timezone is impossible for them?

Reply:
@StackScribe  After looking at the time information from [agenda](https://www.devopsworld.com/agenda), it's quite confusing because some times are shown in UTC/ET/PT/local time so I guess it's better to let the reader figure it out :D

Reply:
@olblak Oy!  I was thinking of just a vague statement -- something like "Sessions are scheduled for ET but some sessions will be repeated at a time that is convenient for people in Asia."  I think I heard that we were doing something like that.  Or just say "Sessions are scheduled for ET but registered attendees can view the recordings, including chat sessions, at a later time."  Just a thought...

Reply:
@StackScribe Hi :). The agenda for the sessions shows the local time i.e when oblak looks up the agenda it will show his local and when I look it up it'll show my (PT) local time. Unfortunately this isn't the same for the workshops. Some workshops shows UTC, some does not. While Olblak and I went through the agenda we felt the way they had it would be easier when individuals look it up themselves vs us trying to document the times which might cause more confusion.